shared/constants/push.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import type {NoErrorTypedAction} from '../constants/types/flux'
+
+export type TokenType = '' | 'ios' | 'android'
```

Comment:
If you are trying to handle a null case, I think it's better to do `TokenType = 'ios' | 'android'` then in a nullable case do `?TokenType`